plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -335,6 +377,18 @@ def inspect(filter, options = {})
       Machinery::Ui.progress(progress)
     end
     Machinery.logger.debug "inspect unmanaged files find calls:#{find_count} files:#{unmanaged_files.size} trees:#{unmanaged_trees.size}"
+
+    processed_files = run_extraction(unmanaged_files, unmanaged_trees, unmanaged_links, excluded_files, remote_dirs,
+      do_extract, file_store_tmp, file_store_final, scope)
+
+    scope.extracted = !!do_extract
+    scope.files = UnmanagedFileList.new(processed_files.sort_by(&:name))
+
+    @description["unmanaged_files"] = scope
+  end
+
+  def run_extraction(unmanaged_files, unmanaged_trees, unmanaged_links, excluded_files, remote_dirs, do_extract,
```

Comment:
Assignment Branch Condition size for run_extraction is too high. [31.97/15]<br>Line is too long. [112/100]